build/ci-build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build Arcus.WebApi CI
+
+trigger:
+  - master
+pr:
+  - master
+
+variables:
+  - group: 'Arcus - GitHub Package Registry'
+  - group: 'Build Configuration'
+
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.105'
+  - name: 'Project'
+    value: 'Arcus.WebApi'
+
+# Building the project:
```

Comment:
Not sure if we need these comments though as the YAML itself serves as living documentation of our build.

What do you think @fgheysels 	?

Reply:
I think they should be removed :)

Reply:
Can you remove them please @stijnmoreels 	? Thanks!